documentation/building/official-build-instructions.md

```diff
@@ -5,20 +5,20 @@ WARNING: These instructions will only work internally at Microsoft.
 
 To kick off an official build, go to this build definition: https://dev.azure.com/dnceng/internal/_build?definitionId=528.
 
-This signs and publishes the following packages to the dotnet feed (currently https://dnceng.pkgs.visualstudio.com/public/_packaging/dotnet5/nuget/v3/index.json):
+This signs and publishes the following packages to the tools feed (https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json):
  - dotnet-dump
+ - dotnet-gcdump
  - dotnet-sos
  - dotnet-trace
  - dotnet-counters
  - dotnet-monitor
- - Microsoft.Diagnostic.TestHelpers
  - Microsoft.Diagnostics.NETCore.Client
 
 To release the latest tools:
 
 1) Merge all the commits for this release from the master branch to the release/stable branch.
 2) Kick off an official build from the release/stable branch.
-3) Change all the package version references in the documentation folder to this official build's package version to maintain the docs up to date. Merge this to both the release and the master branches.
+3) Change all the package version references in the documentation folder to this official build's package version to maintain the docs up to date.
 4) Download the above packages from the successful official build under "Artifacts" -> "PackageArtifacts".
```

Comment:
Isn't your new release tool going to automate some of this?

Reply:
Yes, I will need to update the documentation afterwards. But given that's not merged I haven't

Reply:
Ok